### PR TITLE
updating documentation for new formatting of element attributes

### PR DESCRIPTION
**Pull request by twoseascharlie**
client side documentation update for:
redhat-developer/vscode-xml#633

this PR is dependent on the server side PR:
eclipse/lemminx#1151

### CODE REVIEW

**Comment by twoseascharlie** [comment]
@angelozerr Can you take a look and let me know what you think please?

**Comment by twoseascharlie** [comment]
I've updated my commits with my gpg signature, sorry for the noise

File: docs/Symbols.md [hunk 1]
```diff
@@ -43,21 +43,23 @@ In maven context, the text nodes information are important and we should prefer
...
+`xml.symbols.filters` gives the capability to define filters (to include or exclude) some attributes, some text nodes for a given XML file kind.
 
 Symbols filter are composed with:
 
 * `pattern` (required) : a regular expression matching the file names to which this filter should apply.
-* `expressions` (required) : defines a list of expression. An expression is composed by:
-  *  `xpath` (required) : defines a basic xpath to declare the attribute, the text node which is concerned by the expression.
-  * `excluded` (optional): true if the node which matches the xpath must be excluded or not. By default the excluded is set to false.
-    
+* `expressions` (required) : defines a list of expression. An expression is composed of:
```
Comment (fbricon) [inline docs/Symbols.md:57]: list of expressions

File: docs/Symbols.md [hunk 2]
```diff
@@ -43,21 +43,23 @@ In maven context, the text nodes information are important and we should prefer
...
+  *  `xpath` (required) : defines a basic xpath to declare the attribute or the text node which is concerned by the expression (see [XPath expression](#xpath-expression) below).
+  *  `excluded` (optional): true if the node which matches the xpath must be excluded or not. By default the excluded is set to false.
```
Comment (fbricon) [inline docs/Symbols.md:59]: By default, `excluded` is set to false.

**Review by fbricon — CHANGES_REQUESTED** [review]
Couple minor nitpicks, but looks good overall

File: docs/Symbols.md (hunk 2 above)
Comment (twoseascharlie) [reply docs/Symbols.md]: updated

File: docs/Symbols.md (hunk 1 above)
Comment (twoseascharlie) [reply docs/Symbols.md]: updated

**Comment by angelozerr** [comment]
LGTM @fbricon if you are OK you can merge it.

**Review by fbricon — APPROVED** [review]

**Comment by fbricon** [comment]
thanks @twoseascharlie!

**Pull request merged**